### ISSUE TITLE
3080ti FE nbminer 39.2 Mining ERGO hashrate issue

### ISSUE THREAD

**Issue by liveleeiow** [NebuTech/NBMiner#439]
Hi.

I have a strange issue with my mining setup.

Info as follows:

<img width="629" alt="Screenshot 2021-09-15 at 07 15 41" src="https://user-images.githubusercontent.com/90749229/133382107-566938e1-0147-4967-a9ef-a51bfdc76ee0.png">

OC Settings:

<img width="778" alt="Screenshot-2021-09-13-at-20-37-18" src="https://user-images.githubusercontent.com/90749229/133382371-c89aaea8-88af-4555-893c-9e1156f2a52e.png">
<img width="777" alt="Screenshot-2021-09-13-at-20-37-27" src="https://user-images.githubusercontent.com/90749229/133382374-fcddd008-ad90-4986-a2f5-1a97fcf91d34.png">

I rdp to my rig.  When rdp is connected and i'm watching I get a good hashrate:

<img width="799" alt="Screenshot 2021-09-15 at 07 04 30" src="https://user-images.githubusercontent.com/90749229/133382448-53e79cf9-ee50-4949-8415-e6f116499ba4.png">

As soon as I disconnect from the rdp session the hash rate dramatically drops:

<img width="805" alt="Screenshot 2021-09-15 at 07 05 05" src="https://user-images.githubusercontent.com/90749229/133382461-220de6f8-9fba-48b5-85b2-e0a139d92211.png">

This issue is repeatable and does not appear to be related to uptime at all.

Help!

Thank you.

**Comment by GP-101** [comment]
Could be the same kind of protection as on the 3060's, try letting a screen plugged in to see if the hashrate is still stable after that.

If it's the case buy yourself one of these :

https://www.amazon.fr/EVanlak-Headless-Emulator-Aluminum-Headless-3840x2160/dp/B07YKGGQTJ/ref=asc_df_B07YKGGQTJ

**Comment by liveleeiow** [comment]
Thank you.

After posting I did a little more testing in front of the PC

logged in and screen on - full speed
Screen saver on - lhr 
Turn monitor off - lhr
Load up a game, and leave it active, turn monitor off. - getting 230s, so that’s significantly more than lhr but not 100% max.

I’ve ordered a hdmi dongle.  Fingers crossed this resolves it.

thank you.

**Comment by GP-101** [comment]
Noice, 

So turning the monitor off will always put the LHR lock, but i'm quite surprised that the Screen saver had the same effect.

Didi you manually switched it to screen saver mode or did you let it still until it popped up ?

**Comment by liveleeiow** [comment]
I was in a hurry so limited time to test. I set screen saver to blank and then manually clicked it into preview.

I’ll test later with a normal screen saver that kicks in automatically.

**Comment by liveleeiow** [comment]
Ok, now i'm really confused.  Monitor left turned on, no screen saver eabled.  Look how the hash rate falls away over time here.  Is there some black magic happening somewhere toi slow these down?

[ergolog.txt](https://github.com/NebuTech/NBMiner/files/7170793/ergolog.txt)

**Comment by madsci1016** [comment]
You are hitting LHR lock and it's pulling you back. The extra load of being RDP connected is keeping you from getting caught by the LHR, once you disconnect it get a little more hashing in, trigger LHR, and it pulls you back. bring down core voltage/freq more to keep you under the LHR caught limit. HAd the same issue with my 3060ti and the monitors going to sleep.

**Comment by matrixcoding25** [comment]
I agree with madsci1016 LHR kicks in when core clock is too high on ergo my 3080ti does same thing

**Comment by liveleeiow** [comment]
I've found an OC that's giving me a stable 260 at 210w.

Thanks for advice.
